test/lit/binary/bad-delegate.test

```diff
@@ -0,0 +1,6 @@
+;; Test that we error properly on a file with a bad delegate (a delegate of an
+;; index that does not refer to a any try-catch).
+
+;; RUN: not wasm-opt -all %s.wasm 2>&1 | filecheck %s
+
+;; CHECK: exceptionTargetNames not empty - invalid delegate
```

Comment:
Can we add wast comment to show what wast the wasm was generated from?

Reply:
Good idea, done.